### DISCUSSION TITLE
Issue with OneKey Wallet Installation and Update on Windows 10

### DISCUSSION THREAD

**Discussion by ghost** [OneKeyHQ/app-monorepo#5611] [Q&A] [answered]
Dear OneKey Development Team,

I hope this message finds you well. I am writing to report a bug I encountered with the OneKey Wallet application.

I initially installed version OneKey-Wallet-5.0.1-win-x64.exe on my Windows 10 system, which was configured for "me only." After some time, I received a prompt to update to version OneKey-Wallet-5.1.2-win-arm64.exe.

However, after completing the update, I noticed that the application disappeared from the installation directory and is no longer accessible. I attempted to reinstall the application multiple times, but this did not resolve the issue. Each time, I also cleaned the registry and the installation directories thoroughly. Unfortunately, after performing a clean installation, the only file remaining in the directory \AppData\Local\Programs\OneKey... is uninstall.exe. The wallet client itself is missing and does not appear in any other directories.

I would appreciate your assistance in resolving this issue, as I am eager to use the OneKey Wallet application. Thank you for your attention to this matter.

Best regards

**Comment by ghost** [comment, marked as answer]
![image_2024-08-22_11-01-07](https://github.com/user-attachments/assets/0a36e271-c173-4fd1-b6c6-aec6bb809d44)

Regedit: After removal, this remains in the registry.

**Comment by huhuanming** [comment]
Thank you very much for your feedback. It seems that the issue occurred because the previous registry settings were not successfully overwritten during the repeated installation. We will address this issue in the subsequent versions as soon as possible.
